Editorial changes to MTF Examples (#41)
mulesoft/docs-mule-sdk: modules/ROOT/pages/mtf-expecting-errors.adoc

include::_attributes.adoc[]
endif::[]

> Apart from validating the successful execution of your module's operations, you also must validate the errors that your module throws and the description they generate.

Reviewer: ```suggestion
Apart from validating the successful execution of your module's operations, you also must validate the errors that your module throws and the descriptions they generate.
```

== Prerequesites

> * You are familiarized with how to define the errors your module throws. +

Reviewer: ```suggestion
* You are familiar with how to interpret the errors that your module throws. +
```

Author: `define` here is the correct word. When writing a module, users must define the errors the module throws. It's like defining an exception that a Java class throws.

> }

Reviewer: Line 39, use title case: Operation That Throws Error

----

> You can write tests to validate the expected error type, and error description.

Reviewer: ```suggestion
You can create tests to validate the expected error type and error description.
```

> == Expecting an Error Type

Reviewer: I know that the syntax uses "expect," but it looks to me like maybe we want the term "Validate" instead?
To "expect" is to anticipate, to feel that something is going to happen, but if we are actually testing, then the result of that testing is validating the expectation. Does that make sense?
GLOBALLY evaluate the use of "expect" from here forward; I will not mark each.

Author: "Expecting" an error means validating that your module actually throws those errors. 
I think it would also be correct to say *Validating an Expected Error Type*.

> To expect that the error thrown by the previous module has the proper error type, the test should look as follows:

Reviewer: "You can validate that the error thrown by [what previous module?] has the proper error type as follows:"

> .Expect error type

Reviewer: GLOBALLY standardize this type of heading to use title-style capitals. I will stop marking this from here on.

== Expecting an Error Description

> To expect that the error thrown by the previous module's operation has the proper description, the test should look as follows:

Reviewer: Ditto previous suggested rewording.
